examples/asteroids.rs

```diff
@@ -207,7 +214,9 @@ async fn main() {
             continue;
         }
 
+        // Draw the frame
```

Comment:
I feel like this is a bit misleading? I read it as "clear_background" is going to draw the frame, but it just going to clear background :)

Reply:
I will remove the comments if are misleading for you and pull request again. TY for the comments.

Reply:
Thanks! 
(note that you can just keep committing to this branch and PR will be updated automatically)

Reply:
byt "this" I mean a "master" branch of your fork :)

Reply:
and I ment just one comment, the "draw frame" one, but up to you!